docker/prognostic_run/Dockerfile

```
@@ -2,6 +2,8 @@
 ARG BASE_IMAGE
 FROM ${BASE_IMAGE} as bld
 
+RUN rm /etc/apt/sources.list.d/cuda.list && rm /etc/apt/sources.list.d/nvidia-ml.list || true
```

should we just wait for nvidia to fix their bug?

Also, you can simplify this a bit with `rm -f`. It will exit 0 even if the files don't exist.

```suggestion
RUN rm -f /etc/apt/sources.list.d/cuda.list /etc/apt/sources.list.d/nvidia-ml.list
```

Can you add to `docker/fv3fit/Dockerfile` as well?

---

I think since we don't need to install any nvidia specific stuff, we can just leave this around, but I'll add a comment as to why it's there and that it can be removed if there's a need to apt-get nvidia libraries.